components/chart/templates.md

```diff
@@ -0,0 +1,88 @@
+---
+title: No Data Template
+page_title: Chart - No Data Template
+description: The NoDataTemplate in the Chart for Blazor lets you customize the content displayed when no data is available for any series.
+slug: chart-no-data-template
+tags: telerik,blazor,chart,templates
+published: True
+position: 101
+---
+
+
+# No Data Template
+
+The `NoDataTemplate` allows you to define custom content when any of the Chart series has no data to show. To change the default **No data** localizable text, declare a `<NoDataTemplate>` tag inside a `<ChartSettings>` tag:
```

Comment:
`any` is used in negative statements or questions. Right now, this falls in neither category and `any` sounds unnatural in this instance. Try something like this.

```suggestion
When a Chart series has no data to show, you can use the `NoDataTemplate` to display custom content. This is a global Chart setting and applies to all instances where a Chart series does not have any data to show.

To change the default **No data** localizable text, declare a `<NoDataTemplate>` tag inside a `<ChartSettings>` tag:
```

Adapt these changes to the identical page for the Stock chart as well.

Reply:
Also, please note what's the default behavior somewhere.

Reply:
> Also, please note what's the default behavior somewhere.

The default behavior is mentioned in the next sentence. Is is necessary to note it somewhere else?